strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generation/PackageJson.java

```diff
@@ -0,0 +1,163 @@
+package org.carlspring.strongbox.artifact.generation;
+
+import java.util.ArrayList;
+
+import org.json.JSONArray;
+
+public class PackageJson
+{
+    private String name;
+    private String version;
+    private ArrayList<String> keywords;
+    private ArrayList<License> licenses;
+    private ArrayList<String> contributors;
+    private ArrayList<String> maintainers;
+    private ArrayList<String> files;
+    private ArrayList<String> man;
+    private ArrayList<String> bundledDependencies;
+    private ArrayList<String> os;
+    private ArrayList<String> cpu;
+    private JSONArray dist;
+    
+    public PackageJson() {
+        
+    }
+    public PackageJson(String name, String version, ArrayList<String> keywords, ArrayList<License> licenses,
```

Comment:
This constructor is not a good idea. Each of these values should be set using the respective `set` method.